x/leverage/keeper/keeper_test.go

```diff
@@ -231,22 +189,6 @@ func (s *IntegrationTestSuite) TestSetReserves() {
 	s.Require().Equal(amount, sdk.NewInt(200000000))
 }
 
-func (s *IntegrationTestSuite) TestSetExchangeRate() {
```

Comment:
Note: Removed test as manually setting exchange rates is no longer possible (they are always derived instantly)